CONTRIBUTORS.markdown

```diff
@@ -0,0 +1,50 @@
+Below you can find a list of people that contributed code to the IIS system (in order of appearance) before its move to GitHub (i.e., between September 2012 and September 2015) along with their:
+
+- GitHub login,
+- role in the project,
+- area of the contribution related to IIS source code.
+
+People:
+
+- Mateusz Kobos (mkobos) - team leader, developer
+    - User and technical specification of IIS.
+    - Implementation of some infrastructural elements at the beginning of the project.
+    - Integration of the first [MadIS][]-based module (along with Dominika).
+- Marek Horst (marekhorst) - lead developer
+    - Integration of IIS with OpenAIRE's Information Space (importing data from various data sources and exporting data to Information Space's HBase).
+    - Development of Maven plugin for automated deployment of IIS workflows.
+    - Orchestrating IIS modules using Oozie workflow description and transformer nodes, writing new transformer nodes.
+    - Development of persistent Hadoop cache for [CERMINE][] module. 
+    - Development of functionality of streaming PDFs from Information Space's Object store.
+    - Cooperating with admins on solving Hadoop problems, upgrading, and migrating Hadoop.
+    - Making the code compatible with subsequent versions of Hadoop (CDH3, CDH4, CDH5).
+    - Maintenance of project's build system (Maven projects) - making sure that it works correctly, updating it if necessary.
+    - Maintenance and reorganization of project's code base - removing things that are not needed any more etc.
+        - Extension and reorganization of modules written by other developers (e.g. EuropePMC citation ingestion and parsing).
```

Comment:
Yes, but I think it will be better to just merge these two bullet points into a single one.